consensus/polybft/README.md

```diff
@@ -58,37 +68,37 @@ It has a native support for running bridge, which enables running cross-chain tr
     [--test]
     ```
 
-6. Fund validators on rootchain - in order for validators to be able to send transactions to Ethereum, they need to be funded in order to be able to cover gas cost. **This command is for testing purposes only.**
+7. Fund validators on rootchain - in order for validators to be able to send transactions to Ethereum, they need to be funded in order to be able to cover gas cost. **This command is for testing purposes only.**
 
     ```bash
     $ polygon-edge rootchain fund \
         --addresses 0x1234567890123456789012345678901234567890 \
         --amounts 200000000000000000000
     ```
 
-7. Whitelist validators on rootchain - in order for validators to be able to be registered on the SupernetManager contract on rootchain. Note that only deployer of SupernetManager contract (the one who run the deploy command) can whitelist validators on rootchain. He can use either its hex encoded private key, or data-dir flag if he has secerets initialized:
+8. Whitelist validators on rootchain - in order for validators to be able to be registered on the SupernetManager contract on rootchain. Note that only deployer of SupernetManager contract (the one who run the deploy command) can whitelist validators on rootchain. He can use either its hex encoded private key, or data-dir flag if he has secerets initialized:
 
     ```bash
     $ polygon-edge polybft whitelist-validators --private-key <hex_encoded_rootchain_account_private_key_of_supernetManager_deployer> \
     --addresses <addresses_of_validators> --supernet-manager <address_of_SupernetManager_contract>
     ```
 
-8. Register validators on rootchain - each validator registers itself on SupernetManager. **This command is for testing purposes only.**
+9. Register validators on rootchain - each validator registers itself on SupernetManager. **This command is for testing purposes only.**
 
     ```bash
     $ polygon-edge polybft register-validator --data-dir ./test-chain-1 \
     --supernet-manager <address_of_SupernetManager_contract>
     ```
 
-9. Initial staking on rootchain - each validator needs to do initial staking on rootchain (StakeManager) contract. **This command is for testing purposes only.**
+10. Initial staking on rootchain - each validator needs to do initial staking on rootchain (StakeManager) contract. **This command is for testing purposes only.**
 
     ```bash
     $ polygon-edge polybft stake --data-dir ./test-chain-1 --supernet-id <supernet_id_from_genesis> \
     --amount <amount_of_tokens_to_stake> \
-    --stake-manager <address_of_StakeManager_contract> --native-root-token <address_of_native_root_token>
+    --stake-manager <address_of_StakeManager_contract> --stake-token <address_of_erc20_token_used_for_staking>
```

Comment:
can we obtain the `--stake-manager` from genesis?

Reply:
Yes, but I am not sure we have any command that pulls values from genesis. We do have commands that update the genesis json though. So I've just followed the pattern. Not sure how will deployers and users will use these, and will they have genesis json on the machine they are running the command on, or will they know its location. If it's easier providing the genesis file to the commands, I am all for it, but if it's not easier for them, then maybe it's not worth the effort.

Reply:
I'm okeyish with this for now :)